Documentation/Features/PoA/PoA_set_up_manual.md

```diff
@@ -156,7 +156,7 @@ info: Stratis.Bitcoin.Features.PoA.PoAMiner[0]
       <<==============================================================>>
 ```
 
-to make sure that you are mining.
+To make sure that you are mining. If you are setting the network up from a single node with no connections you need to run the node with `-bootstrap` flag. This flag allows mining when node is in IBD or is not connected to any other node.
```

Comment:
>his flag allows mining when node is in IBD or is not connected to any other node. 

Mining when not connected is only desired when creating a new network (genesis) no? maybe mention this